roles/satellite-clone/files/disassociate_capsules.rb

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/ruby
+username = "admin"
+password = "changeme"
```

Comment:
@sthirugn ok, I understand, you are saying that the commands that are outputted in the logs will have the same username/password (even after they changed it) and there should be a note in the logs, I can change that.